spec/helpers/gitlab_markdown_helper_spec.rb

```diff
@@ -502,25 +504,25 @@ def url_helper(image_name)
     it "should handle references in paragraphs" do
       actual = "\n\nLorem ipsum dolor sit amet. #{commit.id} Nam pulvinar sapien eget.\n"
       expected = project_commit_path(project, commit)
-      markdown(actual).should match(expected)
+      expect(markdown(actual)).to match(expected)
     end
 
     it "should handle references in headers" do
       actual = "\n# Working around ##{issue.iid}\n## Apply !#{merge_request.iid}"
 
-      markdown(actual, {no_header_anchors:true}).should match(%r{<h1[^<]*>Working around <a.+>##{issue.iid}</a></h1>})
-      markdown(actual, {no_header_anchors:true}).should match(%r{<h2[^<]*>Apply <a.+>!#{merge_request.iid}</a></h2>})
+      expect(markdown(actual, {no_header_anchors:true})).to match(%r{<h1[^<]*>Working around <a.+>##{issue.iid}</a></h1>})
```

Comment:
Line is too long. [122/80]<br>Redundant curly braces around a hash parameter.<br>Space missing after colon.<br>Space inside { missing.<br>Space inside } missing.